### PR TITLE
[BF] Quick fix - merge too fast 

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -183,15 +183,17 @@ process Clean_Bundles {
             
         if [ -s "${sid}__\${bname}_cleaned.trk" ]  ; then 
 
-		scil_apply_transform_to_tractogram.py "${sid}__\${bname}_cleaned.trk" \
+        scil_apply_transform_to_tractogram.py "${sid}__\${bname}_cleaned.trk" \
```
Comment (frheault) [inline main.nf]: Below there is one indentation after the else, but none for the if.
I prefer with one.

File: main.nf [hunk 2]
```diff
@@ -183,15 +183,17 @@ process Clean_Bundles {
...
+        scil_compute_streamlines_density_map.py tmp.trk "${sid}__\${bname}_density_mni.nii.gz"
+
+        scil_image_math.py lower_threshold "${sid}__\${bname}_density_mni.nii.gz" 0.01 \
 		    "${sid}__\${bname}_binary_mni.nii.gz"
 		    
-	else "Skipping cleaning for \${bundle} no outliers or all streamlines were outliers"
-	fi
-	
+        else
+            echo "Skipping cleaning for \${bundle} no outliers or all streamlines were outliers"
```
Comment (frheault) [inline main.nf]: This happens only if all are outliers. If no streamlines are considered outliers it saves the file

File: main.nf [hunk 3]
```diff
@@ -183,15 +183,17 @@ process Clean_Bundles {
             
         if [ -s "${sid}__\${bname}_cleaned.trk" ]  ; then
```
Comment (frheault) [inline main.nf]: Maybe a check if the user set the --run_average_bundles to true?

**Comment by SamGuay** [comment]
@arnaudbore, do you think this will soon be merged? Was wondering why it was always stopping abruptly until I discovered the missing `echo` on master that triggered the error. I was about to open a PR but I found this Quick fix PR!

Side comment: Instead of mentioning no outliers OR all streamlines were outliers, that'd be great to know which is applicable for bundle X. That'd make it easier to parse the log to find the "_missing_" bundles in `Clean_bundles/`  since when streamlines are considered outliers the bundle is not saved in this directory and makes it harder to make a list of the missing ones per subject.

**Pull request merged**